contracts/pcv/utils/NamedStaticPCVDepositWrapper.sol

```diff
@@ -0,0 +1,180 @@
+pragma solidity ^0.8.4;
+
+import "../IPCVDepositBalances.sol";
+import "../../Constants.sol";
+import "../../refs/CoreRef.sol";
+import "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+/**
+  @notice a contract to report static PCV data to cover PCV not held with a reliable oracle or on-chain reading 
+  @author Fei Protocol
+
+  Returns PCV in USD terms
+*/
+contract NamedStaticPCVDepositWrapper is IPCVDepositBalances, CoreRef {
+    using SafeCast for *;
+
+    // -------------- Events ---------------
+    event BalanceUpdate(uint256 oldBalance, uint256 newBalance, uint256 oldFEIBalance, uint256 newFEIBalance);
+
+    event DepositRemoved(uint256 index);
+
+    /// @notice struct to store info on each PCV Deposit
+    struct PCVDeposit {
+        string depositName;
+        uint256 usdAmount; /// USD equivalent in this deposit, not including FEI value
+        uint256 feiAmount; /// amount of FEI in this deposit
+        uint256 underlyingTokenAmount; /// amount of underlying token in this deposit
+        address underlying; /// address of the underlying token this deposit is reporting
```

Comment:
for coherence, you could name it `underlyingToken` because you also have `underlyingTokenAmount`. Or rename `underlyingTokenAmount` to `underlyingAmount`

Reply:
fixed